product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Postgres Distributed (PGD) presets on BigAnimal
+navTitle: PGD presets on BigAnimal
+deepToC: true
+---
+
+## Commit scope
+
+[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD is a set of rules that describes the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they impact consistency and reliability.
+
+The actual behavior depends on which kind of commit scope a commit scope's rule uses: [Group Commit](/../../../pgd/latest/durability/group-commit/), [Commit At Most Once](/../../../pgd/latest/durability/camo/), [Lag Control](/../../../pgd/latest/durability/lag-control/), [Synchronous Commit](/../../../pgd/latest/durability/synchronous_commit/), or a combination of these.
+
+Customizing these settings can be complex, presenting a challenge for users. 
+
+### BigAnimal's commit_scope preset options
+
+BigAnimal's commit scope presets simplify managing commit scopes in PGD. These presets balance customization with ease of use, allowing users to enjoy PGD's benefits without the complexity of manual configurations:
+
+| Preset ID | Local group         | External groups     | Use Case                             |
+|-----------|---------------------|--------------------|--------------------------------------|
+| ba001     | MAJORITY, SYNCHRONOUS_COMMIT  | N/A                | High consistency for critical data   |
+| ba002     | ALL, LAG_CONTROL         | ALL, LAG_CONTROL        | Manages replication lag locally and externally |
+| ba003     | MAJORITY, SYNCHRONOUS_COMMIT  | ANY 1, LAG_CONTROL        | Combines high consistency locally with lag control for external|
+| ba004     | MAJORITY, SYNCHRONOUS_COMMIT  | ANY 1, SYNCHRONOUS COMMIT | Higher consistency via external validation |
+
+!!! Note
+ba001 is also the only BigAnimal preset that works with single-grouped PGD clusters. All of them, including ba001, work with multi-grouped clusters.
+!!!
+
+### ba001
+
+ba001 offers a default setup that optimizes data consistency and reliability across distributed environments. Here is an example definition for a group of nodes, `location-a`:
```

Comment:
Execpt none of these commit scopes appear like this.

e.g.

```
bdrdb=> select * from bdr.commit_scopes ;
 commit_scope_id |  commit_scope_name   | commit_scope_origin_node_group |                                                                                commit_scope_rule
-----------------+----------------------+--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
      2651729099 | ba001_p-ghhbcdvrnm-a |                     3588189531 | MAJORITY (p-ghhbcdvrnm-a) SYNCHRONOUS_COMMIT
      3685052456 | ba002_p-ghhbcdvrnm-a |                     3588189531 | ALL (p-ghhbcdvrnm-a) LAG CONTROL (max_commit_delay=1s, max_lag_size=100MB) AND ALL NOT (p-ghhbcdvrnm-a) LAG CONTROL (max_lag_size=1000MB, max_commit_delay=4s, max_lag_time=30s)
       552598214 | ba003_p-ghhbcdvrnm-a |                     3588189531 | MAJORITY (p-ghhbcdvrnm-a) SYNCHRONOUS_COMMIT AND ANY 1 NOT (p-ghhbcdvrnm-a) LAG CONTROL (max_lag_size=1000MB, max_commit_delay=4s, max_lag_time=30s)
      4133704097 | ba004_p-ghhbcdvrnm-a |                     3588189531 | MAJORITY (p-ghhbcdvrnm-a) SYNCHRONOUS_COMMIT AND ANY 1 NOT (p-ghhbcdvrnm-a) SYNCHRONOUS_COMMIT
(4 rows)
```

You may want to add notes about how commit scopes also include the name of the subgroup they are referring to (as this would be the origin node group).

Reply:
I am not following this 100%. Can you expand?

 I took a stab anyway if you want to check it out in the newest push.

Reply:
ba0001 will only be a prefix to a commit scope
The full name of the actual commit scope will be ba001_<subgroupname>

Someone could come away with the expectation of seeing a commit scope called ba001 when there is none.

Reply:
got it.